packages/sdk-core/src/Host.ts

```diff
@@ -40,4 +40,24 @@ export default class Host {
         );
         return new Operation(txn, "SUPERFLUID_CALL_AGREEMENT");
     };
+
+    /**
+     * Creates an Operation of the `callAppAction` function on the host contract.
+     * @param app the address of the Super App you are calling
+     * @param callData the encoded callData for the function
+     * @param overrides ethers overrides object for more control over the transaction sent.
+     * @returns {Operation} an `Operation` class
+     */
+    populateCallAppActionTxnAndReturnOperation = (
```

Comment:
The function name is the whole implementation. That's a bit weird.

Reply:
touché, will rename as: `callAppAction`